.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,47 @@
+name: Build and Deploy
+on: [push]
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+
+      - name: Setup node.js 🏡
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.15.0'
+
+      - name: Cache 📁
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/package-locks.lock') }}
+
+      - name: Install 🔧
+        run: |
+          npm install
+          npm run bootstrap
+
+      - name: Test 🧪
+        run: |
+          npm test
+
+      - name: Build 🏗️
```

Comment:
We don't need to build before we test because we're testing against the TS directly?

Reply:
Right, this PR adds a transform to Jest that allows Jest to do the translating on the fly.

https://github.com/developmentseed/cdk-seed/blob/0b2115115302f11941d636094c08c5e8f36668db/package.json#L28-L31